classes/notification/NotificationDto.php

```diff
@@ -10,7 +12,26 @@ public function __construct(
         public int $created = 0,
         public int $duration = 0,
         public bool $alert = false,
-        public array $attributes = [],
+        protected array $attributes = [],
```

Comment:
```suggestion
```
To go along with previous discussion, I think we should not have this as an actual variable on the NotificationDto class. The `getAttributes()` method below can return a static empty array